frontend/public/components/routes.tsx

```diff
@@ -349,10 +404,17 @@ export type IngressStatusProps = {
   host: string,
   routerName: string,
   conditions: conditionProps[],
-  wildcardPolicy: string
+  wildcardPolicy: string,
+  routerCanonicalHostname: string
 };
 
 export type RouteIngressStatusProps = {
-  ingresses: IngressStatusProps[]
+  ingresses: IngressStatusProps[],
```

Comment:
The type properties in this file end with a `;` instead of `,` (here and throughout)

```suggestion
  ingresses: IngressStatusProps[];
```

Reply:
fixed